src/components/TaskCard.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Button, ButtonVariant } from "./Button";
+import { Task } from "../types/task/Task";
```

Comment:
```suggestion
import { Task } from "../types/Task";
```

Reply:
rename task.ts => Task.ts

Reply:
i used task as a folder to store Task.ts 
Thus did you mean i should rename folder task to Task, right?
Because of Task.ts i already written it with capital first letter.

Reply:
remove folder task. change the file name task.ts to Task.ts